index.js

```diff
@@ -25,6 +57,55 @@ function Changelog( options ) {
 	}
 }
 
+Changelog.ticketParsers = {
+	// Order of keys is important!
+	// When passed several ticketTypes, they will be re-ordered in the order of the keys of this map
+	// This works around github's (or other) regex to be greedy
```

Comment:
The tests revealed github regex matching a JIRA ticket hence so either something is wrong with the regex or it is indeed greedy.

Disable the reordering and run the tests to observe.